src/core_modules/capture-core/components/EventWorkingListsOffline/ColumnSetup/EventWorkingListsOfflineColumnSetup.component.js

```diff
@@ -10,7 +10,8 @@ export const EventWorkingListsOfflineColumnSetup = ({
     customColumnOrder,
     ...passOnProps
 }: Props) => {
-    const defaultColumns = useDefaultColumnConfig(program);
+    const stage = program.stage;
```

Comment:
If the program is a `TrackerProgram` then we need to get the stage with the `stageId`. `program.stage` only works for `EventProgram`. Thanks!

Reply:
This is for Event Program. Thanks!